system/jlib/jstats.cpp

```diff
@@ -2288,6 +2288,23 @@ unsigned __int64 CRuntimeStatisticCollection::getSerialStatisticValue(StatisticK
     return value + convertMeasure(rawKind, kind, rawValue);
 }
 
+void CRuntimeStatisticCollection::set(const CRuntimeStatisticCollection & other, unsigned node)
```

Comment:
These changes look good, but suggest I have got the class structure wrong - I will merge yours first and then go back and fix it....